components/Blog/blog.scss

```diff
@@ -0,0 +1,68 @@
+.blog-page {
+  display: flex;
+}
+
+.side-nav-blog {
+  float: left;
+  align-self: center;
+
+  a {
+    display: block;
+    padding: 15px 25px 15px 100%;
+    color: #c2c2c2;
```

Comment:
for any custom colors that you can't achieve using Tachyons, feel free to save them as SCSS variables.

Reply:
I went through tachyoncs's doc and made few changes... hope it meets the styling guide! :)